config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-kind.yaml

```diff
@@ -603,9 +603,10 @@ presubmits:
         - "cd ./../../k8s.io/kubernetes && ./../../sigs.k8s.io/kind/hack/ci/e2e.sh"
         env:
         - name: FOCUS
-          value: ""
+          value: "."
+        # TODO(bentheelder): reduce the skip list further
         - name: SKIP
-          value: "\\[Disruptive\\]|\\[Flaky\\]|\\[Feature:.+\\]|In-tree.Volumes.\\[Driver:.nfs\\]|PersistentVolumes.NFS|PodSecurityPolicy|Network.should.set.TCP.CLOSE_WAIT.timeout|Mount.propagation.should.propagate.mounts.to.the.host|Simple.pod.should.support.exec.through.an.HTTP.proxy|subPath.should.support.existing.directories.when.readOnly.specified.in.the.volumeSource|ReplicationController.light.Should.scale.from.1.pod.to.2.pods|RuntimeClass.should.run.a.Pod.requesting.a.RuntimeClass.with.a.configured.handler"
+          value: \[Slow\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\]|PodSecurityPolicy|RuntimeClass|LoadBalancer|load.balancer|In-tree.Volumes.\[Driver:.nfs\]|PersistentVolumes.NFS|\[NodeFeature:PodReadinessGate\]|Dynamic.PV|Network.should.set.TCP.CLOSE_WAIT.timeout|Mount.propagation.should.propagate.mounts.to.the.host|Simple.pod.should.support.exec.through.an.HTTP.proxy|subPath.should.support.existing|ReplicationController.light.Should.scale.from.1.pod.to.2.pods|should.provide.basic.identity
```

Comment:
should we document the skips as TODOs?
it will be easier work on them later, right now is only documented in this PR https://github.com/kubernetes-sigs/kind/pull/701/files#diff-d9fa0450190d60ba133fb92282a94725R208

Reply:
once that PR merges link to the explanations?

Reply:
It's not really necessary. None of the other jobs have this and all the presubmit jobs have skip regexes.

The first part is same skip as e2e-gce (except for serial, which we add in the script due to parallel mode), the rest are things that don't work or are flaky. This skip should be changed in a "test" job first (periodic or another manual triggered job) and if it works there this one can be updated.